app/routes/screens/details/Details.js

```diff
@@ -36,7 +34,7 @@ class Details extends Component {
 }
 
 const getAthlete = (state, id) => {
-  if (id !== undefined && state.entities.athletes) {
+  if (id !== undefined && id !== 0 && state.entities.athletes) {
```

Comment:
i don t like that check remove this add inside entities
const nullAthlete = {name: 'not found', ...}
const athletes={
    [undefined]:nullAthlete,
    0:nullAthlete
    423: {name:'bob',...}
}